api/utils/render.js

```diff
@@ -0,0 +1,97 @@
+var puppeteer = require('puppeteer');
```

Comment:
Same here, need JSDoc comments. You can then try generating documentation using `countly docs generate` and view them under yourdomain.com/docs/